qemu/tests/virtual_nic_private.py

```diff
@@ -47,17 +50,22 @@ def data_mon(session, cmd, timeout):
     mon_process_timeout = int(params.get("mon_process_timeout", "1200"))
     sessions = []
     addresses = []
-    vms = []
+    vms = params["vms"].split()
```

Comment:
param['mem'] =  (host_free_memory * 1.2  /  len(vms)) / 64  + 1)  *  64

How about limited guest memory size to 64 times